packages/lesswrong/lib/sql/CreateExtensionQuery.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Builds a Postgres query to create an extension by name.
+ * Note that this doesn't actually extend `Query` like most other query types
+ * as this query has no associated table.
+ */
+class CreateExtensionQuery {
+  constructor(private extensionName: string) {}
+
+  compile(): {sql: string, args: any[]} {
+    return {
+      sql: `CREATE EXTENSION IF NOT EXISTS "${this.extensionName}" CASCADE`,
```

Comment:
The reason I added this is because we now need it in two places. It's definitely not ideal for it not to match the pattern used elsewhere, but both the `IF NOT EXISTS` and `CASCADE` are important enough (and forgettable enough) that I don't really want to risk just duplicating the queries.